client/client-multi/client-c/CMakeLists.txt

```diff
@@ -163,31 +163,19 @@ set(KAA_INCLUDE_DIRS
 
 # Builds Kaa libraries.
 
-set(KAA_BUILD_STATIC kaac_s)
-add_library(kaac_o OBJECT ${KAA_SOURCE_FILES})
-target_include_directories(kaac_o PUBLIC ${KAA_INCLUDE_DIRS})
-
-add_library(${KAA_BUILD_STATIC} STATIC ${KAA_SRC_FOLDER} $<TARGET_OBJECTS:kaac_o>)
-target_link_libraries(${KAA_BUILD_STATIC} PRIVATE ${KAA_THIRDPARTY_LIBRARIES})
-target_include_directories(${KAA_BUILD_STATIC} PUBLIC ${KAA_INCLUDE_DIRS})
-
-if(NOT DEFINED KAA_BUILD_STATIC_ONLY)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
-    set(KAA_BUILD_SHARED kaac)
-    add_library(${KAA_BUILD_SHARED} SHARED ${KAA_SRC_FOLDER} $<TARGET_OBJECTS:kaac_o>)
-    target_include_directories(${KAA_BUILD_SHARED} PUBLIC ${KAA_INCLUDE_DIRS})
-    target_link_libraries(${KAA_BUILD_SHARED} ${KAA_THIRDPARTY_LIBRARIES})
-endif()
+add_library(kaac ${KAA_SOURCE_FILES})
```

Comment:
I meant it:

>  This variable is often added to projects as an OPTION so that each user of a project can decide if they want to build the project using shared or static libraries.